Refresh.Interfaces.APIv3/Endpoints/CategoryApiEndpoints.cs

```diff
@@ -0,0 +1,128 @@
+using AttribDoc.Attributes;
+using Bunkum.Core;
+using Bunkum.Core.Endpoints;
+using Refresh.Core.Configuration;
+using Refresh.Core.Types.Categories;
+using Refresh.Core.Types.Data;
+using Refresh.Database;
+using Refresh.Database.Models.Authentication;
+using Refresh.Database.Models.Levels;
+using Refresh.Database.Models.Users;
+using Refresh.Database.Query;
+using Refresh.Interfaces.APIv3.Documentation.Attributes;
+using Refresh.Interfaces.APIv3.Endpoints.ApiTypes;
+using Refresh.Interfaces.APIv3.Endpoints.ApiTypes.Errors;
+using Refresh.Interfaces.APIv3.Endpoints.DataTypes.Response.Categories;
+using Refresh.Interfaces.APIv3.Endpoints.DataTypes.Response.Levels;
+using Refresh.Interfaces.APIv3.Endpoints.DataTypes.Response.Users;
+using Refresh.Interfaces.APIv3.Extensions;
+
+namespace Refresh.Interfaces.APIv3.Endpoints;
+
+public class CategoryApiEndpoints : EndpointGroup
+{
+    [ApiV3Endpoint("levels"), Authentication(false)]
+    [ClientCacheResponse(86400 / 2)] // cache for half a day
```

Comment:
Lets finally lower this. This was written in early APIv2 where it was assumed categories wouldn't change much or have previews. They still take time to serialize so let's not remove caching entirely.
```suggestion
    [ClientCacheResponse(1800)] // cache for half an hour
```